lib/bridge.js

```diff
@@ -105,6 +109,8 @@ const INTENT_CULL_EVICT_AFTER_MS = 1000 * 60 * 15; // 15 minutes
  * @param {boolean=} opts.roomLinkValidation.triggerEndpoint Enable the endpoint
  * to trigger a reload of the rules file.
  * Default: false
+ * @param {RoomUpgradeHandler~Options} opts.roomUpgradeOpts Options to supply to
+ * the room upgrade  handler. If not defined then no upgrades are performed.
```

Comment:
Upgrades happen regardless of whether or not this is a thing. Instead, can we say "If not defined, the bridge will not handle upgrades"?